_posts/2016-07-04-tsuru-server-1-0-0-is-out.markdown

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+status: publish
+published: true
+title: tsuru server 1.0.0 is out!
+author:
+  display_name: Andrews Medina
+  url: https://github.com/andrewsmedina
+date: '2016-07-04 16:08:00 -0300'
+date_gmt: '2016-07-04 19:08:00'
+categories:
+- Release
+tags:
+- release
+comments: []
+---
+
+tsuru server 1.0.0, along with [tsuru client 1.0.1](https://github.com/tsuru/tsuru-client/releases/tag/1.0.1) and [tsuru-admin 1.0.0](https://github.com/tsuru/tsuru-admin/releases/tag/1.0.0), has been released last week!
+
+This release includes some awesome features and fixes. Please refer to the [release notes](http://docs.tsuru.io/en/stable/releases/tsurud/1.0.0.html) for the full list of features.
+
+Some features worth highlighting are listed below:
+
+* Deploy applications using Docker image
+([#1314](https://github.com/tsuru/tsuru/issues/1314)). Now it’s possible to deploy
+a Docker image as tsuru app using tsuru app-deploy -i command. This image should
```

Comment:
I'd rephrase to "Now it's possible to deploy a Docker image to tsuru using the command ``tsuru app-deploy -i``".